distributed.Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+
+FROM gpuci/miniconda-cuda:$CUDA_VER-devel-$LINUX_VER
+
+ARG CUDA_VER=11.2
+ARG PYTHON_VER=3.8
+ARG RAPIDS_VER=21.08
```

Comment:
If we want the latest nightlies of cudf, shouldn't this be 21.10? Or are we pinning to a specific cuDF version?

Reply:
I can update to `21.10`.